app/src/main/java/com/example/a310_rondayview/CreateEventFragment.java

```diff
@@ -78,30 +105,96 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
```

Comment:
potential splitting into multiple helper functions for better readability. this is not a fatal change but something that would be handy, looks good though!

Reply:
I considered this, but found it hard to split up because of the async methods. Will instead add some comments for now to hopefully explain it better :-)